config/localization.yaml

```diff
@@ -4,6 +4,6 @@ rover_frame: "base_link"
 use_odom_frame: true
 
 gps_linearization:
-  reference_point_latitude: 42.293195
-  reference_point_longitude: -83.7096706
-  reference_point_altitude: 234.1
+  reference_point_latitude: 38.3662183
```

Comment:
probably shouldn't commit this stuff or else we'll immediately have to change it for sim and testing in AA